utils.sh

```diff
@@ -67,6 +67,12 @@ function create_cluster() {
 
     find assets/generated -name '*.yaml' -exec cp -f {} ${assets_dir}/openshift \;
 
+    if [[ "$BMO_WATCH_ALL_NAMESPACES" == "true" ]];
```

Comment:
This will be undefined unless users export BMO_WATCH_ALL_NAMESPACES, so we either need to add a default to e.g `common.sh` or do `${BMO_WATCH_ALL_NAMESPACES:-}` here

Reply:
I had the same trick in #1246, adding to `common.sh` the following should help

```
export BMO_WATCH_ALL_NAMESPACES=${BMO_WATCH_ALL_NAMESPACES:-"false"} 
```

Reply:
defined the default in commons, close to other BMO variables :)